.github/workflows/scripts/kebab-case.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
the power of the generative pre-trained 🤣

Reply:
> the power of the generative pre-trained 🤣

I don't understand

Reply:
too much bash cryptic in this file, it only made sense that it was done by GPT, as a helpful, it's good tho!

Reply:
GPT is the future. Also I made this script lol